install_config/configuring_sdn.adoc

```diff
@@ -199,19 +199,52 @@ This is only supported for {product-title} on Red Hat OpenStack Platform.
 ====
 endif::[]
 
-To enable *flannel* within your {product-title} cluster, set the following
-variables in your Ansible inventory file before running the installation.
+[IMPORTANT]
+====
+Neutron must be disabled, even when security groups are not being used.
```

Comment:
Should be `Neutron port security` here.
From the document, it says `OSP port security controls must be `**enabled**` in Neutron service`. Since I am not familiar with OpenShift-on-OpenStack. Maybe need @jliberma to confirm